src/generator/AutoRest.Ruby.Azure/CodeGeneratorRba.cs

```diff
@@ -66,14 +66,14 @@ public CodeGeneratorRba()
                     continue;
                 }
 
-                if( codeModel.pageModels.Any( each => each.Name.EqualsIgnoreCase(model.Name ) ) )
+                if ( codeModel.pageModels.Any( each => each.Name.EqualsIgnoreCase(model.Name ) ) )
```

Comment:
sorry for the nit-pick here but I don't think this format is the standard we have followed at-least in this code base. 

just one space after `if` and `(`. No spaces bertween `(` & `codeModel` or between `(` & `)`